src/server.js

```diff
@@ -286,10 +286,10 @@ function auth(data) {
 		}
 	} else {
 		client = manager.findClient(data.user, data.token);
-		var signedIn = data.token && data.token === client.config.token;
+		var signedIn = data.token && client && data.token === client.config.token;
```

Comment:
Whaaaat, why did I do that?!

Either way, that other change could have happened as well, I think...